docs/stats/lombscargle.rst

```diff
@@ -415,6 +418,191 @@ For more information on the statistical properties of these normalizations,
+
+For example, let's simulate 60 observations of a sine wave with noise:
+
+>>> t = 100 * rand.rand(60)
+>>> dy = 1.0
+>>> y = np.sin(2 * np.pi * t) + dy * rand.randn(60)
+>>> ls = LombScargle(t, y, dy)
+>>> freq, power = ls.autopower()
+>>> print(power.max())
+0.338140019582
+
+The peak of the periodogram has a value of 0.33, but how significant is
+this peak? We can address this question using the 
+:func:`~astropy.stats.LombScargle.false_alarm_probability` method:
+
+.. doctest-requires:: scipy
+
+  >>> ls.false_alarm_probability(power.max())  # doctest: +FLOAT_CMP
+  0.0043217866919174324
+
+What this tells us is that, under the assumption that there is no periodic
+signal in the data, we will observe a peak this high or higher approximately
+0.4% of the time, which gives a strong indication that a periodic signal is
+present in the data.
+
+.. Note::
+  One must be careful in interpreting this probability: it is a measurement
+  conditioned on the assumption of the null hypothesis of no signal; in symbols,
+  one might write :math:`P({\rm data} \mid {\rm noise-only})`.
+
+  It is tempting to interpret this quantity incorrectly; in particular, you
+  might wish to say a statement like "there is an 0.4% chance that this data
+  is noise only", but this is *not* a correct statement; in symbols, this
+  statement describes the quantity :math:`P({\rm noise-only} \mid {\rm data})`,
+  and in general :math:`P(A\mid B) \ne P(B\mid A)`.
+
+  See [11]_ for a more detailed discussion of such caveats.
+
+We might also wish to compute the required peak height to attain any given
+false alarm probability, which can be done with the
+:func:`~astropy.stats.LombScargle.false_alarm_level` method:
+
+.. doctest-requires:: scipy
+
+  >>> probabilities = [0.1, 0.05, 0.01]
+  >>> ls.false_alarm_level(probabilities)  # doctest: +FLOAT_CMP
+  array([ 0.25446627,  0.27436154,  0.31716182])
+
+This tells us that to attain a 10% false alarm probability requires the highest
+periodogram peak to be approximately 0.25; 5% requires 0.27, and 1% requires
+0.32.
+
+False Alarm Approximations
+--------------------------
+
+Although the false alarm probability at any particular frequency is analytically
+computable, there is no closed-form analytic expression for the more relevant
+quantity of the false alarm level of the *highest* peak in a particular
+periodogram.
+This must be either determined through bootstrap simulations, or approximated
+by various means.
+
+AstroPy provides four options for approximating the false alarm probability,
+which can be chosen using the ``method`` keyword:
+
+- ``method="baluev"`` (the default)
+  implements the approximation proposed by Baluev 2008 [8]_,
+  which employs extreme value statistics to compute an upper-bound of the false
+  alarm probability for the alias-free case. Experiments show that the bound is
+  also useful even for highly-aliased observing patterns.
+
+  .. doctest-requires:: scipy
```

Comment:
>I'm very confused by the CircleCI error: it's failing saying scipy is not installed, on a doctest that is wrapped in .. doctest-requires:: scipy.
>
> Any ideas?

@jakevdp  You need to remove the leading spaces before `.. doctest-requires:: scipy`.  This is why your doctests are failing.

Reply:
But the doctest in question is part of a bulleted list... Won't that end the list if I remove these spaces?

Reply:
Perhaps not... There is similar usage in https://raw.githubusercontent.com/astropy/astropy/bdb4ec936fa789a24db28d0329e4a66ed10892ec/docs/convolution/kernels.rst

Reply:
OK – I'll give it a try!